client/src/main/java/com/paypal/selion/configuration/ListenerInfo.java

```diff
@@ -48,8 +66,18 @@ public boolean isEnabled() {
         return enabled;
     }
 
+    /*
+     * Returns boolean value of the JVM argument when available, else returns the system default.
+     */
     static boolean getBooleanValFromVMArg(String vmArgValue) {
-        boolean flag = true;
+        return getBooleanValFromVMArg(vmArgValue, defaultIsEnabled);
+    }
+
+    /*
+     * Returns boolean value of the JVM argument when available, else returns the {@code isEnabledWhenJvmAbsent}.
+     */
+    static boolean getBooleanValFromVMArg(String vmArgValue, boolean isEnabledWhenJvmAbsent) {
+        boolean flag = isEnabledWhenJvmAbsent;
```

Comment:
@krmahadevan, I did not get this comment. I have renamed member of `class ListenerInfo` and the method parameters to its readability better.